corehq/apps/userreports/reports/view.py

```diff
@@ -356,6 +356,10 @@ def excel_response(self):
         return file
 
 
+class CustomConfigurableReport(ConfigurableReport):
+    slug = 'custom_configurable'
```

Comment:
whenever i see a subclass that only overrides members I feel like there must be a better way to use instances and parameterization. (just a musing thought - not really sure if it makes sense here)